src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -289,8 +302,23 @@ export const SecurityAnalysisResultTab: FunctionComponent<
                                 <FormattedMessage id="ContingenciesFromConstraints" />
                             </MenuItem>
                         </Select>
-                    </Box>
-                )}
+                    )}
+                    {(tabIndex === NMK_RESULTS_TAB_INDEX ||
+                        tabIndex === N_RESULTS_TAB_INDEX) && (
+                        <SecurityAnalysisExportButton
+                            studyUuid={studyUuid}
+                            nodeUuid={nodeUuid}
+                            csvHeaders={csvHeaders}
+                            resultType={resultType}
+                            disabled={
+                                !result ||
+                                (result.content &&
+                                    result.content.length === 0) ||
+                                result.length === 0
```

Comment:
If there are existing results the button isn't disabled while a new computation is running, maybe using `securityAnalysisStatus` or `isLoadingResult` could help